db/models.py

```diff
@@ -17,7 +21,7 @@ def __str__(self):
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
Is it correct? Should I make Meta Class with index?

Reply:
in my opinion it's enough )